### ISSUE TITLE
F-Droid can't build

### ISSUE THREAD

**Issue by licaon-kter** [Taewan-P/gpt_mobile#16]
ref: https://gitlab.com/fdroid/fdroiddata/-/jobs/7274011735#L509

fix: https://github.com/mikepenz/AboutLibraries/issues/784#issuecomment-1205501876

/LE: fyi https://gitlab.com/fdroid/fdroiddata/-/commit/2b9a224936f9c920b0c674c0c2a3cdd026bc520a

**Comment by Taewan-P** [comment]
@licaon-kter Sorry! Just fixed at #17 and released a new version: [v0.4.1](https://github.com/Taewan-P/gpt_mobile/releases/tag/v0.4.1)!

**Comment by Taewan-P** [comment]
Should be working by now. The local hash and CI build hash matches

**Comment by licaon-kter** [comment]
```
2024-07-05 15:33:27,068 ERROR: Could not build app dev.chungjungsoo.gptmobile: compared built binary to supplied reference binary but failed
==== detail begin ====
verification of APK with copied signature failed
Comparing reference APK to APK with copied signature...
Unexpected diff output:
==== detail end ====
```

this reminds me of: https://gitlab.com/fdroid/fdroiddata/-/issues/3299#note_1985568674

looking at https://github.com/Taewan-P/gpt_mobile/blob/v0.4.1/.github/workflows/release-build.yml#L9

and confirmed by https://github.com/actions/runner-images/commit/d31e0a18cbe4583142e746fc692570b706db0192#diff-46b808a5c93af22ed67b614d1f4070b549bebe5f8b13714e72f134ca58b1965eR248

you are using build-tools 35 to sign, so to fix reproducibility make this: https://github.com/Taewan-P/gpt_mobile/blob/v0.4.1/.github/workflows/release-build.yml#L52 say `apksigner sign --alignment-preserved true  ...`

**Comment by Taewan-P** [comment]
@licaon-kter How about now? I recreated the `v0.4.1` tag with the updated workflow, and the APK from it

**Comment by linsui** [comment]
It works, thanks!

**Comment by Taewan-P** [comment]
Oh... I see. There is no way that I can use this library then?

**Comment by Taewan-P** [comment]
Just released [v0.6.1](https://github.com/Taewan-P/gpt_mobile/releases/tag/v0.6.1) with removing the library. Can you check if the build works?

**Comment by licaon-kter** [comment]
Thanks :+1:
